addons/medical_feedback/functions/fnc_handleEffects.sqf

```diff
@@ -58,4 +58,9 @@ if ((!GVAR(heartBeatEffectRunning)) && {_heartRate != 0} && {(_heartRate > 160)
 [!_unconscious, _pain] call FUNC(effectPain);
 [!_unconscious, _bleedingStrength, _manualUpdate] call FUNC(effectBleeding);
 
+// - Tourniquets, fractures and splints indication ---------------------------------------
+if (GVAR(GUIIndication)) then {
```

Comment:
Not a fan of naming it `GUIIndication`, I think something like `HUDIndicators` would be more adequate. But that's just a suggestion.